repair-service/rest-app/src/main/java/com/epam/repair/rest_app/RestApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.repair.rest_app;
```

Comment:
bad name of the package `rest_app`
I personally haven't ever seen names with `_` in the name

Reply:
`rest` enough

Reply:
Comment accepted. Package name changed to rest.